woocommerce/woocommerce-android
# Move localize.rb script to plugin

This PR removes the localize.rb script from the repository and adds it to the `release-toolbox` plugin, so that it can be shared and reused in the WordPress project. 

### To Test:
1. Run `bundle install`
2. Run `bundle exec fastlane localize_libs` and verify that the script is executed without errors and no changes are made.
3. Make some changes to the `strings.xml` file of the Android Login Library, run `bundle exec fastlane localize_libs` again and verify that the changes are merged into the app `strings.xml`.
